sdcm/nemesis.py

```diff
@@ -3612,6 +3613,9 @@ def _disrupt_network_block_k8s(self, list_of_timeout_options):
         time.sleep(15)
         self.cluster.wait_all_nodes_un()
 
+    @decorate_with_context_if_issues_open(
+        ignore_coredump_on_network_block_disruption,
+        issue_refs=['https://github.com/scylladb/scylla-cluster-tests/issues/9135'])
```

Comment:
since the filter is very exact, not sure we need to pin it on a issue that we'll need to keep open till forever ?

Reply:
consider backported releases, once we'll close this issue, might still fail.